tron/mesos.py

```diff
@@ -362,6 +383,23 @@ def submit(self, task):
 
         mesos_task_id = task.get_mesos_id()
         self.tasks[mesos_task_id] = task
+        env = task.get_config()['environment']
+        clusterman_resource_str = env.get('CLUSTERMAN_RESOURCES')
+        clusterman_metrics, _ = get_clusterman_metrics()
+        if clusterman_resource_str and clusterman_metrics:
+            clusterman_resources = json.loads(clusterman_resource_str)
+            aws_region = env.get('EXECUTOR_AWS_REGION')
```

Comment:
I am probably forgetting the details of a previous review, but how are callers going to set this? I am wondering if it would make sense to do something similar to what we do in spark-run where we look up the AWS region from the cluster (https://github.com/Yelp/paasta/blob/799cc1da422c6c7f53fe1d91e0eb9e0dc6d6ec3f/paasta_tools/cli/cmds/spark_run.py#L578)

Reply:
Fixed